Documentation/TransferringFiles.md

```diff
@@ -30,3 +30,60 @@ sudo mount -t ext2 _disk_image mnt
 Some archiving tools, like [7-Zip](https://www.7-zip.org/), are capable of directly opening ext2 images like Serenity's `_disk_image`. With these, you can open the disk image like any other archive and extract the files you need.
 
 For WSL users: If you have the image on your native WSL drive (recommended), this drive can be opened in Explorer by manually opening `\\wsl$` (not visible in the Network tab!) and then the "network share" corresponding to your distro.
+
+## Method 4: Enable OpenSSH and use sftp
+
+In Windows 10, if you have not yet already done so, you can enable OpenSSH server:
```

Comment:
The method it self isn't specific to `Windows 10`, but this guide is written that way. 

I think it would be better if we wrote the documentation to be generic instructions on how to install OpenSSH on serenity, and then how to transfer files in/out.

We should either omit instructions for how to set up OpenSSH on the readers host, or link to existing resources (ex: https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse). Those steps are not really relevant to the project, and there are so many possible options and variations that readers could be using, we can't possibly enumerate them all in our documentation.

Reply:
Will do just that, thanks for giving relevant viewpoint on how best to approach this case.

Reply:
Wait, we should or shouldn't use links to existing resources? If not, then I must remove the link I used in the document. I used this -> https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse .

Reply:
What you have seems fine